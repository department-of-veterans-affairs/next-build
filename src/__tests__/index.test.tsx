import { render } from '@testing-library/react'
import HomePage from '../pages/index'
import '@testing-library/jest-dom'

// HomePage renders without crashing
test('HomePage renders without crashing', () => {
  render(<HomePage nodes />)
})

// shallow render homepage with nodes
test('HomePage renders a list of nodes', () => {
  const { container } = render(<HomePage nodes={[{ title: 'Test' }]} />)
  expect(container.firstChild).toMatchSnapshot()
})
