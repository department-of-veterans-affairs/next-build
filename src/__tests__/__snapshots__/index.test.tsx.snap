// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders a list of nodes 1`] = `null`;
